iocore/net/UnixNetAccept.cc

```diff
@@ -38,6 +38,28 @@ safe_delay(int msec)
   socketManager.poll(nullptr, 0, msec);
 }
 
+static int
+drain_throttled_accepts(NetAccept *na)
+{
+  struct pollfd afd;
+  Connection con[THROTTLE_AT_ONCE];
+
+  afd.fd     = na->server.fd;
+  afd.events = POLLIN;
+
+  int n = 0;
+  while (check_net_throttle(ACCEPT, Thread::get_hrtime()) && n < THROTTLE_AT_ONCE - 1 && socketManager.poll(&afd, 1, 0) > 0) {
```

Comment:
A for loop would be cleaner and you want `n < THROTTLE_AT_ONCE` not `n < THROTTLE_AT_ONCE -1`